src/main/java/it/gov/pagopa/pu/workflow/wf/receiptclassification/iuf/config/IufReceiptClassificationWfConfig.java

```diff
@@ -0,0 +1,35 @@
+package it.gov.pagopa.pu.workflow.wf.receiptclassification.iuf.config;
+
+import io.temporal.workflow.Workflow;
+import it.gov.pagopa.payhub.activities.activity.classifications.*;
+
+import it.gov.pagopa.pu.workflow.config.BaseWfConfig;
+import it.gov.pagopa.pu.workflow.config.TemporalWFImplementationCustomizer;
+import it.gov.pagopa.pu.workflow.wf.receiptclassification.iuf.helper.TransferClassificationWfHelperActivity;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+
+@Configuration
+@ConfigurationProperties(prefix = "workflow.iuf-receipt-classification")
+public class IufReceiptClassificationWfConfig extends BaseWfConfig {
```

Comment:
just call it `IufClassification`
```suggestion
public class IufClassificationWfConfig extends BaseWfConfig {
```